lra/coordinator/server/src/main/resources/application.yaml

```diff
@@ -28,7 +28,7 @@ helidon.lra.coordinator:
     connection:
       url: "jdbc:h2:file:./lra-coordinator"
       username: "h2"
-      password: ""
+      password: ${EMPTY}
```

Comment:
This is not correct for version 2.x, we do not understand the ${EMPTY} replacement

Reply:
fixed